Tweak Norwegian Bokmål translation

cristeahub opened the pull request:
This tweak is minor and fixes some grammatical errors.

I did not change `kopier` to `kopiér`, even though that is a common way to write the word. The language council of Norway suggest writing the word as it is written here, without the accent, but it does read weird and allows for misunderstanding. If this was my project I would have added the accent. Let me know if you want the accent added.

What do you think regarding the accents @Uzaaft ?

cristeahub commented on po/nb_NO.UTF-8.po:
```diff
@@ -191,21 +192,21 @@ msgstr "Hovedmeny"
 "⚠️ You're running a debug build of Ghostty! Performance will be degraded."
 msgstr ""
-"⚠️ Du kjører en debug-bygg av Ghostty. Debug-bygg har redusert ytelse."
+"⚠️ Du kjører et debug-bygg av Ghostty. Debug-bygg har redusert ytelse."
```
We currently use the proposed wording internally at my work, i.e. "bygg" for "build", and I have seen it having widespread usage in talks and events. If you feel "build" is better we can change to it, but it would still be neuter, and we would keep the hyphen.

So we would end up with this
```suggestion
"⚠️ Du kjører et debug-build av Ghostty. Debug-build har redusert ytelse."
```

We could even go crazy and ignore the hyphen and write it as one word, but I think that would cause more confusion than not, since build and debug is not that common used in our language.

Uzaaft commented:
I feel like the accent is just a bit archaic and most tech people wouldn't use it, and probably prone to be outdated at some points, so I am a happy to just keep what we have atm.  @cristeahub (Also what happened to your discord user 😢 )

pluiedev commented:
There's a merge conflict (new lines were added that need translating). Could you also translate those while you're here? Thanks :D

cristeahub commented:
Yes, will fix

cristeahub committed:
Tweak Norwegian Bokmål translation

This tweak is minor and fixes some grammatical errors.

I did not change `kopier` to `kopiér`, even though that is a common way
to write the word. The language council of Norway suggest writing the
word as it is written here, without the accent, but it does read weird
and allows for misunderstanding. If this was my project I would have
added the accent. Let me know if you want the accent added.

cristeahub commented on po/nb_NO.UTF-8.po:
```diff
@@ -37,17 +38,19 @@ msgstr "OK"
 "One or more configuration errors were found. Please review the errors below, "
 "and either reload your configuration or ignore these errors."
 msgstr ""
+"Én eller flere konfigurasjonsfeil ble funnet. Vennligst gjennomgå feilene under, "
```
Just since I'm quite the language nerd and mentioned accents. This word means the number 1, while `en` is an article. So here it means "one or more", instead of "a or more".

Uzaaft replied:
You god damn nerd😂But you are right.

Uzaaft reviewed (approved):
This looks much better to me :D Appreciate your language nerdiness :P
